### ISSUE TITLE
Gitcoin Grants Discovery is too centralized; Build something could get huge usage by help us decentralize it.

### ISSUE THREAD

**Issue by owocki** [gitcoinco/skunkworks#195]
# TLDR 
![4s5ox1](https://user-images.githubusercontent.com/513929/103422106-fd14b800-4b5c-11eb-964a-f39e9c9c7c2e.jpg)

# Background

We recently ran an a/b test on the Gitcoin Grants default landing page, and found evidence that [GItcoin can affect which grants get funded by changing its discoverability tools](https://twitter.com/owocki/status/1340058876440244225).  

Shit.  Gitcoin doesn't want this power or this responsibility!  Gitcoin wants to remove ourselves from building opinionated software about what grants should be quadratic funded, and focus more on building a strong checkout experience, and [a powerful data engine that detects collusion](https://twitter.com/block_science/status/1341809547170615296), and greater awareness of QF's capabilities among matching partners.

Vitalik, in the [Gitcoin Grants Round 8 Wrap up Livestream](https://youtu.be/nQ71Xb0a2FE?t=958), actually suggested

> INSERT_VITALIK_QUOTE_HERE

So we that's where we need your help, community & friends.  We want to decentralize Gitcoin Grants Discovery, by empowering the community to extend Gitcoin Grants!  

We want you to be a part of that!

# Bounty Prompt

The prompt for this hack is as follows: "There are 1300 (and growing) GItcoin Grants.  What is the best way to discover Grants?
"

The details are up to you as you build the tool.

Here's the plan:
0. Gitcoin provides a series of [APIs](https://github.com/gitcoinco/web/blob/master/docs/API.md) that (a) a GET API that allows you to pull an inventory of Gitcoin Grants (b) a POST API allow users to checkout a *CART* (a cart is a list of (grant_id, amount_to_contribute, token_to_contribute_in) tuples ) and a *REFERRAL_ADDRESS* - an ETH address that you own
1. You - build a tool that utilizes this API, to provide a better Gitcoin Grants Discoverability experience than anyone else.
2. We'll pay you to build this, and you could also get your `REFERRAL_ADDRESS` funded if we upgrade the checkout experience for round 9 to allow users that you send through Gitcoin Grants, to fund your grant. 

Here's what we're thinking:

<img width="1502" alt="Screen Shot 2020-12-31 at 11 52 48 AM" src="https://user-images.githubusercontent.com/513929/103422458-cf307300-4b5e-11eb-9ec5-571d62523486.png">

# Rewards

We will be posting this as a contest bounty to Gitcoin, where we'll fund 

1. One Top Prize - 4k DAI
2. One Second Prize - 1.5k DAI
3. Up to 3 Honorable mentions - 700 DAI each

The deadline to submit a working demo of your discoverability experience is 1/31/2021, in time for Gitcoin Grants Round 9 (mid February 2021).

# API info

To get a list of grants to put into your app, 
```
curl https://gitcoin.co/grants/grants.json
```

or to pull a specific offset or limit from this inventory, use 
```
curl https://gitcoin.co/grants/grants.json?offset=X,limit=Y
```

To send a user to checkout with a cart of grants, send them to.
```
https://gitcoin.co/grants/cart/bulk-add/grant_1_id;grant_1_amount_to_fund;grant_1_token_id,grant_2_id;grant_2_amount_to_fund;grant_2_token_id/?REFERRAL_ADDRESS=your_address
# note: DAI is token_id:283
# example URL: https://gitcoin.co/grants/cart/bulk-add/238;5;283,468;5;283
```

**Comment by owocki** [comment]
Some ideas from a few friends. @hudsonjameson

# A choice-based architecture 

<img width="773" alt="Screen Shot 2021-01-08 at 11 18 23 PM" src="https://user-images.githubusercontent.com/513929/104084852-38238500-5208-11eb-9795-14ceaf1a9461.png">

# A swipe left / swipe right type interface that learns your preferences

<img width="750" alt="Screen Shot 2021-01-08 at 11 17 42 PM" src="https://user-images.githubusercontent.com/513929/104084838-1a562000-5208-11eb-8e32-b5d0b76edb24.png">

# Grant of the Day


<img width="701" alt="Screen Shot 2021-01-08 at 11 17 30 PM" src="https://user-images.githubusercontent.com/513929/104084848-2e9a1d00-5208-11eb-9d07-46bec7197ae5.png">

**Comment by owocki** [comment]
API work https://github.com/gitcoinco/skunkworks/issues/212
